proseco/tests/test_catalog.py

```diff
@@ -497,7 +502,7 @@ def test_bad_pixel_dark_current():
     assert np.all(aca.acqs.dark[-245 + 512:512, 454 + 512] == ACA.bad_pixel_dark_current)
 
     exp_ids = [2, 100, 101, 102, 103]
-    assert sorted(aca.guides['id']) == exp_ids
+    assert sorted(aca.guides['id']) == sorted(exp_ids + [4])
```

Comment:
It might be more self-documenting in the test to just have exp_ids_acq and exp_ids_guide, but I think you prefer fewer variables.

Reply:
My bar for code quality / readability is way lower in tests (both in writing and reviewing).

Reply:
All good.  Just when doing the light code review here my process was ... "wait, why isn't id=4 in exp_ids?  Oh it probably isn't a good acq star (would guess it is too close to that bad pixel that is the point) and he uses exp_ids as the acq stars one line down".